File: 03_Wednesday/prac.js
function getString() {
    stringOne = "hello"
    stringTwo = "pillow"

    return stringOne && stringTwo
}


function logic(stringOne, stringTwo) {
    stringOneList = []
    stringTwoList = []

    i = 0
    similarity = ""
    for (s in stringOne) {
        stringOneList.push(s)
    }
    for (s in stringTwo) {
        stringTwoList.push(s)
    }

    while (i < stringOneList.length && i < stringTwoList.length) {
        if (stringOneList[i] != stringTwoList[i]) {
            similarity += "."
        } else {
            similarity += stringOneList[i]
        }
        i += 1
        return similarity
    }
}

function work() {
    stringOne = getString()
    stringTwo = getString()
    console.log(logic(stringOne, stringTwo))
}

work()

File: 03_Wednesday/prac.py
def get_string():
    string = input("Insert String: ")
    return string


def logic(string_one, string_two):
    string_one_list = []
    string_two_list = []
    i = 0
    similarity = ""
    for s in string_one:
        string_one_list.append(s)
    for s in string_two:
        string_two_list.append(s)

    while (i < len(string_one_list) and i < len(string_two_list)):
        if (string_one_list[i] != string_two_list[i]):
            similarity += "."
        else:
            similarity += string_one_list[i]
        i += 1

    return similarity


def main():
    string_one = get_string()
    string_two = get_string()
    print(logic(string_one, string_two))


if __name__ == '__main__':
    main()
